src/index.ts

```diff
@@ -9,7 +9,7 @@ import * as commons from './commons';
 
 import { Response as MachineLoginResponse } from './routes/authentication/machine-login';
 
-const apiVersion = 8;
+const apiVersion = 9;
```

Comment:
Is this still a correct change? If I understand the current state of the pull request correctly we are enforcing the stricter rules on add and update routes regardless of api version. That makes me wonder what breaking change the 9 version contains that the 8 version does not.

Reply:
Not being able to send localised names longer than 255 chars, it's still a breaking change I think

Reply:
Yes it is breaking but it is breaking regardless of the apiVersion that you send it with. If someone sends us a long string with a apiVersion 4 then the server still validates against the length and gives an error.

Reply:
75937ee